rocksdb_admin/utils.h

```diff
@@ -41,4 +41,9 @@ bool EncodeThriftStruct(const T& obj, std::string* data) {
   return !ex;
 }
 
+// Will remove and create a directory
+// returns true if any of these operations failed.
```

Comment:
should we return `false` upon failure? `true` usually indicates success

Reply:
alternatively, how about we just return the error message, and when it succeeds, it'll be empty.